CDTDatastoreCLEncrpytedTest/main.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  main.swift
+//  CDTDatastoreCLEncrpytedTest
```

Comment:
This has a typo in the name `CDYDatastoreCLEncr**py**tedTest`. There appears to be a duplicate with the correct name `CDTDatastoreCLEncryptedTest/main.swift`